### DISCUSSION TITLE
"Verified" badge is not visible on my organization's profile page

### DISCUSSION THREAD

**Discussion by MrDeveloperJIS** [community/community#61264] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![github-ictflow-issue3](https://github.com/community/community/assets/31595886/f4898f9d-1303-4a2c-88ac-56e48ace0df5)
![github-ictflow-issue1](https://github.com/community/community/assets/31595886/9ecb6303-9821-48d7-90a0-b099f9f97da9)
![github-ictflow-issue2](https://github.com/community/community/assets/31595886/33301f2e-f205-44a3-9311-7326b223fe82)
![image](https://github.com/user-attachments/assets/1af6fe6a-d7a3-4103-abfe-5ac39307eff5)

The "Verified" badge is not visible on my organization's profile page.

**Comment by 50bvd** [comment]
If the "Verified" badge is not appearing on your organization's profile page on GitHub, there are a few potential reasons why this might be the case. Here are a few troubleshooting steps to take:

Confirmation Email: Make sure you have confirmed the email associated with your organization. GitHub often sends a confirmation email when you create an account or make significant changes. If you haven't confirmed your email, GitHub might not verify your organization.

Public Email: GitHub will only display the verified badge for the organization if the email address is public. If your organization's email is private, it might not display the "Verified" badge.

Email Domain Match: For an organization, the "Verified" badge appears when the organization’s website matches the domain of a confirmed email for the organization.

Verification Status: It's possible that GitHub has not yet verified your organization. You may need to contact GitHub support to find out if there are additional steps you need to take.

Waiting Period: After confirming your email, there might be a delay before the "Verified" badge appears. Try waiting for a bit to see if it shows up.

If none of the above steps solve your problem, then contacting GitHub support might be your best bet. They can provide specific help based on your exact situation and help solve any issues you may be encountering.

**Comment by MrDeveloperJIS** [comment]
As you can see, I have provided a domain-based public email. I verified my domain "ictflow.com" and provided the website link "https://www.ictflow.com" on my data. I didn't receive any confirmation emails at all. I have no idea what the issue is. Please help me to solve the issue.

**Reply by rehborn** [reply]
@MrDeveloperJIS you have verified the domain for github-pages only (not necessary for the verification badge), you need to verify it for the organization itself, it's located at:

> Organization -> Settings (section Security) -> Verified and approved domains

for reference: [docs](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization#verifying-a-domain-for-your-organization)

**Comment by Cayth-X** [comment]
Nice

**Comment by byrneh** [comment]
@MrDeveloperJIS follow instructions here exactly [Verifying a domain for your organization](https://docs.github.com/en/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization#verifying-a-domain-for-your-organization)
